var/spack/repos/builtin/packages/py-setproctitle/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySetproctitle(PythonPackage):
+    """The setproctitle module allows a process to change its title (as
+    displayed by system tools such as ps and top)."""
+
+    homepage = "https://github.com/dvarrazzo/py-setproctitle"
+    url      = "https://pypi.io/packages/source/s/setproctitle/setproctitle-1.1.10.tar.gz"
+
+    version('1.1.10', sha256='6283b7a58477dd8478fbb9e76defb37968ee4ba47b05ec1c053cb39638bd7398')
+
+    depends_on('python', type=('build', 'run'))
```

Comment:
You can remove this line, it's already added by PythonPackage

Reply:
Good point, thanks! Guess I got confused because it was proposed by the new package template and I didn't check the PythonPackage class explicitly. I have created another PR (#15173) to clarify this.